python/lsst/daf/butler/core/quantum.py

```diff
@@ -93,11 +94,25 @@ def actualInputs(self):
         """
         return self._actualInputs
 
+    @property
+    def outputs(self):
+        """A `dict` of output datasets (to be) generated for this quantum,
+        with the same form as `predictedInputs`.
+
+        Read-only; update via `addOutput()`.
```

Comment:
I think you may well need two backticks here for numpydoc because it (and `predictedInputs` text above) are not classes. I think the alternative is to write something like `~lsst.daf.core.butler.Quantum.addOutput()` (or whatever); ie give a fully qualified method name and the `~` means to truncate the output to the final element. I haven't tested this though.

Reply:
BTW, I almost literally copied docstring from other method in this class, should I change it everywhere?

Reply:
Ah. Leave it then. I'll fix it next time I try to run the doc gen and see all the warnings appear.